docs/editors/eclipse.md

```diff
@@ -0,0 +1,27 @@
+---
+id: eclipse
+title: Eclipse
+---
+
+Metals works with Eclipse thanks to the
+[`metals-eclipse`](https://github.com/scalameta/metals-eclipse) plugin.
+
+![Eclipse demo](https://i.imgur.com/jrmjVf0.gif)
```

Comment:
GIF tip: I recommend trying to minimize the screen size and minimize the length. In this particular case, it might be enough to demonstrate a single completion + single diagnostic. Heavy GIFs make the website slower to load. Our landing page + overview pages are already too fat 😅

Reply:
I will try to minimize that.

Reply:
Should be better :)